### ISSUE TITLE
Improve weak alternative text

### ISSUE THREAD

**Issue by unclesam2024** [web-infrastructure/doc_architecture#40]
[Alternative text](https://support.silktide.com/glossary/alternative-text/) should be descriptive and useful. Alternative text shown here appears to be of insufficient value.

This check will highlight alternative text that appears to be a filename, a URL, machine generated code, or otherwise not of value to a real person.
